sections/shorting/ManageShort/ManageShortAction.tsx

```diff
@@ -533,7 +533,8 @@ const ManageShortAction: FC<ManageShortActionProps> = ({
 								gasPrices={gasPrices}
 								feeReclaimPeriodInSeconds={0}
 								quoteCurrencyKey={null}
-								feeRate={feeRate}
+								totalFeeRate={feeRate}
```

Comment:
Shorts doesn't have dynamic fees yet. Can you disable dynamic fees from appearing under shorts?

Reply:
Thanks. The shorting page only shows ```feeRate``` if ```feeRate``` equals ```totalFeeRate```. In this case, ``feeRate``` always equals ```totalFeeRate``` so it's disabled dynamic fees naturally. Do you mean we should add new logic to disable dynamic fee on shorting page?

Reply:
I see, that should be fine. 

I was only concerned because it looks we're adding new props into the shorts components. Do we need to pass through baseFeeRate here?

Reply:
Yes. It passes ```totalFeeRate``` and ```baseFeeRate``` but the values of them come from ```issueFeeRate```.

Reply:
Yes, we can. It requires some logic changes in *FeeRateSummaryItem* because now the *baseFeeRate* always shows first and then compare it with *totalFeeRate* to see if we need to show dynamic fee.

Reply:
Okay up to you which fee prop to make optional. I just don't think we should pass in two props with the same data type.

Reply:
Yes. I see your points. The change has been made. Now the ```baseFeeRate``` is optional. We don't pass any value into shorting.